components/page-header/src/index.js

```diff
@@ -80,5 +80,5 @@ PageHeader.defaultProps = {
 
 export default PageHeader;
 
-export asLogoAnchor from './hoc/asLogoAnchor';
-export asNavAnchor from './hoc/asNavAnchor';
+export LogoAnchor from './atoms/logo-anchor';
```

Comment:
Export these as static objects on the root component

So they can be used e.g.

```jsx
<PageHeader>
  <PageHeader.NavAnchor>...
```